src/infra/db-stateful-set.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: db
```

Comment:
@amerkul consider better naming for your PostgreSQL Stateful set

what is the db?
name must be related somehow to the type of database that you are using